content/en/docs/distributions/nutanix/install-kubeflow.md

```diff
@@ -106,20 +106,128 @@ Add the following  under staticPasswords section
+  If not, you can install the [MetalLB](https://metallb.universe.tf/) load balancer manifests on your Karbon cluster.
+  ```
+  $ kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.10.2/manifests/namespace.yaml
+  $ kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.10.2/manifests/metallb.yaml
+  ```
+
+  After the manifests have been applied, we need to configure MetalLB with the IP range that it can use to assign external IPs to services of type LoadBalancer. You can find the range from the subnet in Prism Central’s [networking and security](https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Flow-Networking-Guide:ear-flow-nw-view-subnet-list-pc-r.html) settings.
+  ```
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    namespace: metallb-system
+    name: config
+  data:
+    config: |
+      address-pools:
+        - name: default
+          protocol: layer2
+          addresses:
+          - <IP_ADDRESS_RANGE: x.x.x.x-x.x.x.x>
+  ```
+  Create a ConfigMap with the following information, substitute the addresses field with your IP address range, and apply it to the cluster.
+  ```
+  $ kubectl apply -f metallb-configmap.yaml
+  ```
+
+## Access Kubeflow Central Dashboard
+There are multiple ways to acces your Kubeflow Central Dashboard:
+- Port Forward: The default way to access Kubeflow Central Dashboard is by using Port-Forward. You can port forward the istio ingress gateway to local port 8080.
     
    ```
    kubectl --kubeconfig=<karbon_k8s_cluster_kubeconfig_path> port-forward svc/istio-ingressgateway -n istio-system 8080:80
    ```
     
-You can now access the Kubeflow Central Dashboard at http://localhost:8080. At the Dex login page, enter user credentials that you previously created.
+  You can now access the Kubeflow Central Dashboard at http://localhost:8080. At the Dex login page, enter user credentials that you previously created.
     
  
-For accessing through NodePort, you need to configure HTTPS. Create a certificate using cert-manager for your Worker node IP in your cluster. Add HTTPS to kubeflow gateway as given in [Istio Secure Gateways](https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/). Then access your cluster at
+- NodePort: For accessing through NodePort, you need to configure HTTPS. Create a certificate using cert-manager for your Worker node IP in your cluster. Add HTTPS to kubeflow gateway as given in [Istio Secure Gateways](https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/). Then access your cluster at
    
    ```
    https://<worknernode-ip>:<https-nodeport>
    ```
+- LoadBalancer: If you have a LoadBalancer set up (See optional "Setup a LoadBalancer" section above), you can access the dashboard using the external IP by making the following changes.
+  - Update Istio Gateway to expose port 443 with HTTPS and make port 80 redirect to 443:
```

Comment:
is this newly added here? we had only a link pointing to all this https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/. 
Why is this needed to be documented here as compared to the link earlier.

Reply:
as discussed with @thunderboltsid, we will file a separate tracker and separate out the https part which is common to https and loadbalancer section.